src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1659,7 +1662,19 @@ export class SearchView extends ViewPane {
 			if (typeof node === 'string') {
 				dom.append(span, document.createTextNode(node));
 			} else {
-				const link = this.instantiationService.createInstance(Link, node);
+				const link = this.instantiationService.createInstance(Link, node, {
+					opener: async href => {
+						const parsed = URI.parse(href, true);
+						if (parsed.scheme === Schemas.command) {
+							const result = await this.commandService.executeCommand(parsed.path);
+							if ((result as any)?.triggerSearch) {
+								this.triggerQueryChange();
```

Comment:
What command are you going to use this for?

Reply:
`remoteHub.enableIndexing`